pkgs/development/node-packages/node-env.nix

```diff
@@ -444,11 +443,6 @@ let
         # Run post install hook, if provided
         runHook postInstall
       '';
-
```

Comment:
Is this deletion intentional?

Reply:
The generation script does not create it. It shouldn't be commited.